### ISSUE TITLE
ONNX model makes different predictions for AveragedPerceptronBinaryClassifier and LinearSvmBinaryClassifier

### ISSUE THREAD

**Issue by ganik** [microsoft/NimbusML#435]
repro
`
from nimbusml.datasets import get_dataset
from nimbusml.linear_model import AveragedPerceptronBinaryClassifier
from nimbusml.preprocessing import OnnxRunner

iris_df = get_dataset("iris").as_df()
iris_df.drop(['Species'], axis=1, inplace=True)
iris_no_label_df = iris_df.drop(['Label'], axis=1)
iris_binary_df = iris_no_label_df.rename(columns={'Setosa': 'Label'})

predictor = AveragedPerceptronBinaryClassifier()
predictor.fit(iris_binary_df)
print(predictor.predict(iris_binary_df))
predictor.export_to_onnx("test.onnx", 'com.microsoft.ml')
onnxrunner = OnnxRunner(model_file="test.onnx")
print(onnxrunner.fit_transform(iris_binary_df))
`

Output

0      1.0
1      1.0
2      1.0
3      1.0
4      1.0
      ...
145    0.0
146    0.0
147    0.0
148    0.0
149    0.0
Name: PredictedLabel, Length: 150, dtype: float64
     Sepal_Length  Sepal_Width  Petal_Length  ...  PredictedLabel.onnx.0  Score.onnx.0  Probability.onnx.0
0             5.1          3.5           1.4  ...                    0.0     -0.312534            0.642229
1             4.9          3.0           1.4  ...                    0.0     -0.323769            0.637826
2             4.7          3.2           1.3  ...                    0.0     -0.295820            0.648736
3             4.6          3.1           1.5  ...                    0.0     -0.321254            0.638814
4             5.0          3.6           1.4  ...                    0.0     -0.304930            0.645196
..            ...          ...           ...  ...                    ...           ...                 ...
145           6.7          3.0           5.2  ...                    0.0     -1.764737            0.131769
146           6.3          2.5           5.0  ...                    0.0     -1.573707            0.173585
147           6.5          3.0           5.2  ...                    0.0     -1.630237            0.160219
148           6.2          3.4           5.4  ...                    0.0     -1.755978            0.133483
149           5.9          3.0           5.1  ...                    0.0     -1.509973            0.189693

[150 rows x 25 columns]

**Comment by pieths** [comment]
Superseded by https://github.com/microsoft/NimbusML/issues/438
